Doc/library/string.rst

```diff
@@ -259,9 +259,9 @@ on the value, ``'!r'`` which calls :func:`repr` and ``'!a'`` which calls
 
 Some examples::
 
-   "Harold's a clever {0!s}"        # Calls str() on the argument first
-   "Bring out the holy {name!r}"    # Calls repr() on the argument first
-   "More {!a}"                      # Calls ascii() on the argument first
+   "Harold's a clever {0!s}"        # Calls str() on the first argument
+   "Bring out the holy {name!r}"    # Calls repr() on a named argument 
+   "More {!a}"                      # Calls ascii() on an implicit argument 
```

Comment:
The examples say "on the argument first", because it's trying to say that `!a`, `!s`, and `!r` are conversions that are called first. That is, they're called before the resulting string is formatted.

I agree that the comments aren't great, but I think any changes need to make this point about conversions being applied first, and your edits lose that. Maybe something like "Calls str() on the argument before formatting the resulting string"? I realize it's a little long.

Reply:
Okay Eric, that is not how I read it when I encountered it. I can see what you mean about the original. This change is unnecessary then.